src/main/kotlin/me/zeroeightsix/kami/feature/module/Nuker.kt

```diff
@@ -45,15 +45,19 @@ object Nuker : Module() {
 
     private val range
         get() =
-            // this doesn't look that great, but i can't think of a cleaner way to do this
             if (usePlayerRange) mc.interactionManager?.reachDistance?.toDouble() ?: hitRange
             else hitRange
 
+    // This exists to use the range without having to recalculate `range` a lot.
+    // Instead it is calculated once at the start of the tick event
+    private var currentRange = range
+
```

Comment:
Oh god, no.
Please just use a local variable and pass it on through parameters.

Reply:
alright then, i'll change that. i'm trying my best